vtex-apps/store-header
# Feat/add fixed handle

#### What is the purpose of this pull request?

- Fix linting and formatting issues.
- Add a `--fixed` to the `stickyRow` when it sticks to the top of the window.

~It's a WIP proof of concept, I want to talk to @lbebber first.~

#### What problem is this solving?

Styling headers with css.

#### How should this be manually tested?

Add multiple sticky header rows to your `store-theme` and scroll them away to see the `--fixed` modification class being toggled.

https://kiwi--storecomponents.myvtex.com/

#### Screenshots or example usage

#### Types of changes
